vault/ha.go

```diff
@@ -46,6 +46,8 @@ const (
 	// leaderPrefixCleanDelay is how long to wait between deletions
 	// of orphaned leader keys, to prevent slamming the backend.
 	leaderPrefixCleanDelay = 200 * time.Millisecond
+
+	haAllowedMissedHeartbeats = 2
```

Comment:
Is 2 arbitrary?

Reply:
Yeah, it is 😬

Reply:
Seems fine? We can always adjust later if we need to I guess.